.github/workflows/check-pr.yaml

```diff
@@ -0,0 +1,48 @@
+name: Check PR
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
[memo]
I added `concurrency` based on the following description. (so that if multiple workflows are executed on the same branch, the oldest one can be automatically cancelled)


https://github.com/yumemi-inc/flutter-mobile-project-template/blob/bf2aab61e9519cb20b5bd15663bd16107d4bed51/.github/workflows/check-pr.yaml#L6-L8